src/components/tabs/tabs.scss

```diff
@@ -259,7 +262,7 @@ md-tab {
   font-size: 14px;
   text-align: center;
   line-height: $tabs-header-height - 24;
-  padding: 12px 24px;
+  padding: 12px;
```

Comment:
@mmalerba this is one other change that is likely to cause screenshot diffs. The result is that there is more room in the tab to show the text and less chance of it being truncated. This also better aligns with the spec.

Are you seeing screenshot diffs that are actually breaking changes or are they just slightly shifted text within the tabs?

Reply:
The changes looked mostly ok, but due to the number of screen diffs I'm not really sure we can make the change in google3

Reply:
I tried removing this padding change, but it doesn't work without it. Those two CSS changes are the key to fixing this P2 Regression that was introduced in 1.1.2.

Here's what it looks like w/o the padding change:
![tabs-pagination-ie11-broken2](https://user-images.githubusercontent.com/3506071/46046872-aeac1f80-c0f0-11e8-8318-45390ef4d5d3.PNG)